src/AlgebraicGeometry/Schemes/ProjectiveSchemes/Objects/Attributes.jl

```diff
@@ -485,7 +485,7 @@ homogeneous_coordinate_ring(P::ProjectiveScheme) = P.S
 ### type getters
 projective_scheme_type(A::T) where {T<:AbstractAlgebra.Ring} = projective_scheme_type(typeof(A))
 projective_scheme_type(::Type{T}) where {T<:AbstractAlgebra.Ring} =
-ProjectiveScheme{T, mpoly_dec_ring_type(mpoly_ring_type(T))}
+proj{T, mpoly_dec_ring_type(mpoly_ring_type(T))}
```

Comment:
This is a type getter and should therefore return the type.
```suggestion
ProjectiveScheme{T, mpoly_dec_ring_type(mpoly_ring_type(T))}
```